src/pages/market/market.ts

```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+import { NavController } from 'ionic-angular';
+import { Storage } from "@capacitor/core";
+
+@Component({
+  selector: 'page-market',
+  templateUrl: 'market.html'
+})
+export class MarketPage {
+  karma: string
+
+  constructor(public navCtrl: NavController) {
+    Storage.get({key: 'karma'}).then((resp) => {
+      this.karma = resp.value
+    })
+  }
+
+  sell(){
```

Comment:
The class method 'sell' must be marked either 'private', 'public', or 'protected'<br>block is empty<br>missing whitespace